futaba/cogs/moderation/core.py

```diff
@@ -241,6 +244,7 @@ async def self_jail(self, ctx, minutes: int = 60):
             )
 
             await self.perform_jail(ctx, member, minutes, "Self jail")
+            await self.perform_mute(ctx, member, minutes, "Self jail")
```

Comment:
@Scrub000 I've tested it and it looks like there is an error when it tries to mute.
Error:
```[ERROR] [2020/05/26 23:51:50] futaba.client: Unexpected error during command!
Traceback (most recent call last):
  File "C:\Users\maware\Documents\Projects\Python\futaba\venv\lib\site-packages\discord\ext\commands\core.py", line 83, in wrapped
    ret = await coro(*args, **kwargs)
  File "C:\Users\maware\Documents\Projects\Python\futaba\futaba\cogs\moderation\core.py", line 247, in self_jail
    await self.perform_mute(ctx, member, minutes, "Self jail")
TypeError: perform_mute() takes 4 positional arguments but 5 were given
```

Screenshot:
![image](https://user-images.githubusercontent.com/18365785/82958005-493ac100-9fac-11ea-84fd-bb85520a4fd1.png)